pkg/test/integration/common/framework.go

```diff
@@ -94,6 +94,13 @@ var (
 
 	// if true, means that the tags present on VM are strings not key-value pairs
 	isTagsStrings = os.Getenv("TAGS_ARE_STRINGS")
+
+	// if true, control cluster is a seed
+	// only set this variable if operating in gardener context
+	isControlSeed = os.Getenv("IS_CONTROL_CLUSTER_SEED")
+
+	// To detect orphans with this tag
+	clusterTagSuffix = "integration-test-cluster"
```

Comment:
```suggestion
	targetClusterPlaceholder = "integration-test-cluster"
```
Make it a constant instead of variable